static/js/components/RecentSources.jsx

```diff
@@ -63,7 +63,7 @@ export const useSourceListStyles = makeStyles(() => ({
     fontSize: "1rem",
   },
   sourceTime: {
-    color: "gray",
+    color: theme.palette.info.main,
```

Comment:
```suggestion
    color: theme.palette.text.secondary,
```
I found these quite hard to see in light mode (looks good on dark mode though!) because the info.main is a light color against the white background:
![Screenshot 2021-02-09 130034](https://user-images.githubusercontent.com/17696889/107407039-266b2100-6ad7-11eb-8cda-ee3fe0fa5d5f.png)
I'd suggest using the text color as those adjust based on "light" or "dark" type for the palette.

Reply:
I didn't catch this one, but I agree, this isn't great. I'll update!